Documentation/RunningTests.md

```diff
@@ -0,0 +1,76 @@
+## Running SerenityOS Tests
+
+There are two classes of tests built during a Serenity build: host tests and target tests. Host tests run on the build
+machine, and use [Lagom](https://github.com/SerenityOS/serenity/blob/master/Meta/Lagom/README.md) to build Serenity
+userspace libraries for the host platform. Target tests run on the Serenity machine, either emulated or bare metal.
+
+### Running Host Tests
+
+There are two ways to build host tests: from a full build, or from a Lagom-only build. The only difference is the CMake
+command used to initailize the build directory.
+
+For a full build, pass `-DBUILD_LAGOM=ON` to the CMake command.
+
+```sh
+mkdir Build
+cd Build
+cmake .. -GNinja -DBUILD_LAGOM=ON
+```
+
+For a Lagom-only build, pass the Lagom directory to CMake.
+
+```sh
+mkdir BuildLagom
+cd BuildLagom
+cmake ../Meta/Lagom -GNinja
+```
+
+In both cases, the tests can be run via ninja after doing a build:
+
+```sh
+ninja && ninja test
+```
+
+### Running Target Tests
+
+Tests built for the serenity target get installed either into `/usr/Tests` or `/bin`. `/usr/Tests` is preferred, but
+some system tests are installed into `/bin` for historical reasons.
+
+The easiest way to run all of the known tests in the system is to use the `run-tests-and-shutdown.sh` script that gets
+installed into `/home/anon/tests`. When running in CI, the environment variable `$DO_SHUTDOWN_AFTER_TESTS` is set, which
+will run `shutdown -n` after running all the tests.
+
+For completeness, a basic on-target test run will need the serenity image built and run via Qemu.
```

Comment:
nitpick:

```suggestion
For completeness, a basic on-target test run will need the serenity image built and run via QEMU.
```

Reply:
Caps applied throughout to Serenity and QEMU